### PR TITLE
CIV-12798 - Dashboard Notification - When HWF Full remission is granted (Claimant)

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12798

### Change description ###
Added dashboard notification for Claimant when HWF application for full remission is granted.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/ClaimIssueHwfFullRemissionGrantedScenarioTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import uk.gov.hmcts.reform.civil.model.citizenui.HelpWithFeesDetails;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+
+import java.math.BigDecimal;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Testcontainers
```
Comment (dharmendrak) [inline src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/ClaimIssueHwfFullRemissionGrantedScenarioTest.java:23]: this is moved to DashboardBaseIntegration

**Review by laarmada — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public String camundaActivityId(CallbackParams callbackParams) {
     private CallbackResponse notifyApplicantForHwFOutcome(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
         CaseEvent hwfEvent = caseData.getHwFEvent();
-        if (Objects.nonNull(caseData.getApplicant1Email())) {
+        if (Objects.nonNull(caseData.getApplicant1Email()) && CaseEvent.FULL_REMISSION_HWF != hwfEvent) {
```
Comment (dharmendrak) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java:57]: why this change is here, we wouldn't know background why we did this here

Comment (neeta-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java:58]: as discussed added a 1line comment explaining the reason.

**Review by dharmendrak — APPROVED** [review]
